### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by michaelbeil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-We will set up [Node](http://nodejs.org/) (JavaScript), [Python](http://www.python.org/), [CPlusPlus](http://www.cplusplus.com/), and [Ruby](http://www.ruby-lang.org/) environments. Even if you don't program in all three, it is good to have them as many command-line tools use one of them. We also install a few daily use application and Latex. As you read and follow these steps, feel free to send me any feedback or comments you may have.
+We will set up [Node][] (JavaScript), [Python][], [CPlusPlus][], and [Ruby][] environments. Even if you don't program in all four, it is good to have them, as many command-line tools use one of them. We also install a few daily use applications and Latex. As you read and follow these steps, feel free to send me any feedback or comments you may have.
 
 All contributions to the book are welcome. Please help add support for other libraries and languages.
 
 **Note:** This book has been generated using [GitBook](http://www.gitbook.io) and is open source, feel free to contribute or signal issues on [GitHub](https://github.com/sb2nov/mac-setup).
+
+[Node]: http://nodejs.org
+[Python]: http://www.python.org
+[CPlusPlus]:(http://www.cplusplus.com
```
Comment (sb2nov) [inline README.md]: random bracket here I think.

Comment (michaelbeil) [inline README.md]: oh whoops!

**Comment by michaelbeil** [comment]
Should be set now.

**Comment by sb2nov** [comment]
Thanks for the changes :+1:

**Comment by michaelbeil** [comment]
You bet! :+1: 

Thanks for writing this book! :100:

**Pull request merged**